docs/content/v1/usage/lifecycle.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Lifecycle callbacks
+weight: 8
+---
+
+LEGO comes with 5 methods that allows you to hook some callbacks within the component lifecycle.
+
+```js
+export default class extends Lego {
+  init() { … }
+
+  connected() { … }
+
+  rendered() { … }
+
+  changed() { … }
+
+  disconnected() { … }
+}
+```
+
+## `init`
+
+This callback is called on the first time when the element is added to the DOM. At this point the element is still not rendered, so you can't access the [Shadow DOM]((https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM)) and inner elements.
```

Comment:
Should add a `(see Accessing the component's DOM)` link to the script documentation.